.github/workflows/docker.yml

```
@@ -0,0 +1,94 @@
+name: Docker
+
+on:
+  push:
+    branches: [ master ]
+  
+jobs:
+  build:
+    name: Build and test Waltz - postgresql
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.6
+        env:
+          POSTGRES_USER: waltz
+          POSTGRES_DB: waltz
+          POSTGRES_PASSWORD: waltz
```

The parameters should be the same as in `.build.settings.xml` instead.

```suggestion
          POSTGRES_USER: postgres
          POSTGRES_DB: waltz
          POSTGRES_PASSWORD: postgres
```